Java11 problem with log4j2

## Bug Report

### Affected tool(s) or class(es)
Any tool that uses log4j2 and is compiled in java11

### Affected version(s)
- [ X ] Latest master branch as of 20210707

### Description 
Runing almost everything (anything that makes use of log4j2) I get an UnsuportedOperationException. For example:

```
java -Dsamjdk.use_async_io_read_samtools=false -Dsamjdk.use_async_io_write_samtools=true -Dsamjdk.use_async_io_write_tribble=false -Dsamjdk.compression_level=2 -Xmx120g -XX:ParallelGCThreads=20 -jar /opt/gatk-4.2.0.0-42-g2fc3b65-SNAPSHOT/gatk-package-4.2.0.0-42-g2fc3b65-SNAPSHOT-local.jar HaplotypeCaller -R Reference/Cork_oak_ref.fasta -I 03-Mapping/05-Deduped-Sorted/S_B_10-deduped-sorted.bam -O 04-SNPcalling/01-HaplotypeCaller/S_B_10.raw.g.vcf --emit-ref-confidence GVCF
WARNING: sun.reflect.Reflection.getCallerClass is not supported. This will impact performance.
Exception in thread "main" java.lang.ExceptionInInitializerError
at org.broadinstitute.hellbender.Main.extractCommandLineProgram(Main.java:304)
at org.broadinstitute.hellbender.Main.setupConfigAndExtractProgram(Main.java:180)
at org.broadinstitute.hellbender.Main.mainEntry(Main.java:202)
at org.broadinstitute.hellbender.Main.main(Main.java:289)
Caused by: java.lang.UnsupportedOperationException: No class provided, and an appropriate one cannot be found.
at org.apache.logging.log4j.LogManager.callerClass(LogManager.java:576)
at org.apache.logging.log4j.LogManager.getLogger(LogManager.java:601)
at org.apache.logging.log4j.LogManager.getLogger(LogManager.java:588)
at org.broadinstitute.barclay.argparser.ClassFinder.<clinit>(ClassFinder.java:29)
... 4 more
```

#### Steps to reproduce
Compile master with java11 and run 
`gatk CheckIlluminaDirectory --help`

#### Expected behavior
Normal execution

#### Actual behavior
UnsupportedOperationException

--- a/build.gradle
+++ b/build.gradle
@@ -429,7 +429,8 @@ tasks.withType(Jar) {
                 'Main-Class': project.mainClassName,
                 'Picard-Version': picardVersion,
                 'htsjdk-Version': htsjdkVersion,
-                'Spark-Version': sparkVersion
+                'Spark-Version': sparkVersion,
+                'Multi-Release': 'true'
     }
 }
 

